live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -323,3 +323,37 @@
   with_items:
     - { key: 'kernel.core_pattern', value: '/var/crash/core.%e.%p.%t' }
     - { key: 'vm.overcommit_memory', value: '1' }
+
+#
+# Add the apt repos that have the debug versions of packages, so that we
+# can download a debug kernel to use with crash.
+#
+- apt:
+    name: 'ubuntu-dbgsym-keyring'
+- apt_repository:
+    repo: "{{ item }}"
+  with_items:
+    - deb http://ddebs.ubuntu.com bionic main restricted universe multiverse
```

Comment:
This isn’t wrong, but I think it might be more consistent to add these in a file in “live-build/base/config/archives” .. then we could add the “ubuntu-dbgsym-keyring” package to the minimal package list file, right?

Would that work?

Reply:
We need `ubuntu-dbgsym-keyring` installed before we add these repositories, otherwise we will get an error when we try to fetch the list of packages from the repositories:
```
GPG error: http://ddebs.ubuntu.com bionic Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY C8CAB6595FDFF622
```
I think this will prevent us from adding these repositories in “live-build/base/config/archives”. We could probably move `ubuntu-dbgsym-keyring` into the minimal package list if we wanted though.

Reply:
Would that work? adding `ubuntu-dbgsym-keyring` to the minimal package list work?

Reply:
I’d like to do it that way, since that’ll be note consistent with how we’re adding other APT sources, if it works. Can you try that and let me know?

Otherwise this change looks good.